### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by Ukaypromise**
### Adding Event Handlers

- [ ] Add logic files for calculating and operation.
- [ ] Add events handlers for click operations.
- [ ] Fix linter errors.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,18 +1,167 @@
...
+          >
+            -
+          </button>
+          <button
+            type="button"
+            onClick={this.handleClick}
+            className="btn btn-Div"
+          >
+            /
+          </button>
```
Comment (AmaduKamara) [inline src/components/Calculator.js]: - [ ] The "/" sign you are using on this button is not the one passed for division in the `operate.js` file and for that reason when you want to perform division, the button text content does not match the one passed in the condition in the operate.js file and hence the app crashes to a blank screen. Kindly edit the division sign in the operate.js file to have the "/" sign. 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Ukaypromise, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR
✅ README is professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 1 above)
Comment (Ukaypromise) [reply src/components/Calculator.js]: I just figured it out after I submitted it for review. It has been fixed @AmaduKamara

**Review by Lordkaito — APPROVED** [review]
Hi @Ukaypromise, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**